README.rst

```diff
@@ -812,6 +812,11 @@ Choose from the list of available rules:
 
   The PHP constants ``true``, ``false``, and ``null`` MUST be in lower case.
 
+  Configuration options:
+
+  - ``case`` (``'lower'``, ``'upper'``): whether to use the ``upper`` or ``lower`` case
```

Comment:
so, `lowercase_constants` will make sth uppercase ?
we shall rename in BC way

Reply:
What do you propose then?

Reply:
@drupol 

For an example, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.15/src/Fixer/Whitespace/BlankLineBeforeStatementFixer.php.

Reply:
@localheinz Thanks for the example, but I don't understand what you're trying to show me :-)

Reply:
I think, what is meant is that the name of the rule `lowercase_constants`  feels odd for something that might make things upper case. The rename in BC form would;
- think of a new rule name, like `constants_case` for example
- create the file for the fixer by that name and copy paste everything from the current fixer you got here to the new file
- change the class name etc. in that file so it matches and passes
- (do the same for the test files of the fixer)
- than turn the current fixer into a proxy fixer, which directs calls on it to the new one (sample of a proxy fixer is [MethodSeparationFixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.12.7/src/Fixer/ClassNotation/MethodSeparationFixer.php) which proxies to ClassAttributesSeparationFixer)

I can help do all of that. However I think first we need to get some more :+1:  's to prevent doing all that work while not making the merge.

Reply:
Thanks @SpacePossum , I've implemented everything in my last commit and did a rebase as well.

Reply:
All the tests are passing except on Travis, it seems that it's unrelated to this PR.